gammapy/irf/edisp/core.py

```diff
@@ -52,6 +52,14 @@ class EnergyDispersion2D(IRF):
     required_axes = ["energy_true", "migra", "offset"]
     default_unit = u.one
 
+    @property
+    def _default_offset(self):
```

Comment:
Note that this might have to evolve once we introduce more complex IRF with FoV spatial dependency